libbeat/publisher/pipeline/controller.go

```diff
@@ -65,15 +67,16 @@ func newOutputController(
 	b queue.Queue,
 ) *outputController {
 	c := &outputController{
-		beat:     beat,
-		monitors: monitors,
-		observer: observer,
-		queue:    b,
+		beat:      beat,
+		monitors:  monitors,
+		observer:  observer,
+		queue:     b,
```

Comment:
that's funny. Why is the parameter named `b` ? As we've modified this piece of code, leats clean up the naming a little :)

Reply:
Fixed in 5ff7219.